src/sentry/search/events/constants.py

```diff
@@ -13,7 +13,7 @@
 ISSUE_ALIAS = "issue"
 ISSUE_ID_ALIAS = "issue.id"
 RELEASE_ALIAS = "release"
-SEMVER_ALIAS = "sentry.semver"
+SEMVER_ALIAS = "release.version"
```

Comment:
![image](https://user-images.githubusercontent.com/1421724/125334631-73f6ee80-e300-11eb-81c4-8e018eb4d414.png)

This appears to be used in a bunch of places on the frontend already. Do we need to update all of those?

Reply:
Yep, @scttcper will do that in a separate PR